service/src/main/java/uk/gov/hmcts/reform/fpl/enums/GeneratedEPOKey.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.fpl.enums;
```

Comment:
Do you think it would be useful to move this into a further sub package `ccdfields`? Maybe a refactor for the future?

Reply:
Refactor in the future as this would should cover Sam's CMOkeys also.